crypto/x509/x_all.c

```diff
@@ -41,18 +41,26 @@ int NETSCAPE_SPKI_verify(NETSCAPE_SPKI *a, EVP_PKEY *r)
 
 int X509_sign(X509 *x, EVP_PKEY *pkey, const EVP_MD *md)
```

Comment:
> I made one more commit for the functions in x_all.c.

Good. I just suggest re-wording the message of the 2nd commit to, e.g,. the single line
```
X509 x_all.c: Set 'modified' flag when ASN1_item_sign{,_ctx} call was successful.
```